core/templates/services/image-upload-helper.service.ts

```diff
@@ -60,6 +60,7 @@ export class ImageUploadHelperService {
       return blob;
     }
 
+    // Return null when the image cannot be converted.
```

Comment:
@leylatuon We generally add these sort of comments at top of the functions, also I think below comment  would be more appropriate
```
 // Image file returned will be null when blob is not of type
 // 'image' or blob size is zero.
```

Reply:
@mridul-netizen I see, that is good to know, thank you!